relayer/relayer/src/connections/azero.rs

```diff
@@ -0,0 +1,179 @@
+use std::sync::{Arc, Mutex};
```

Comment:
Why change the directory layout so much, `signer` could just live in a top level, just like the `relayer` did?
This will cause massive refactors for opened PRs

Reply:
I wanted to make it a workspace so that they have common dependency versions.